packages/orbit-components/src/Card/index.tsx

```diff
@@ -68,7 +77,12 @@ const Card = ({
   };
 
   return (
-    <StyledCard spaceAfter={spaceAfter} data-test={dataTest} id={id}>
+    <StyledCard
+      // TODO: remove spaceAfter in the next major version
```

Comment:
Should we to document that `spaceAfter` is deprecated?

Reply:
better will be to remove that from READMEs

Reply:
Isn't it possible to add some linter warning when using the prop? Just removing from the docs seems a bit too silent